modules/src/core/ics24_host/validate.rs

```diff
@@ -67,11 +67,19 @@ pub fn validate_port_identifier(id: &str) -> Result<(), Error> {
     validate_identifier(id, 2, 128)
 }
 
+/// Default validator function for Channel identifiers.
+///
+/// A valid Identifier must be between 10-64 characters and only contain lowercase
+/// alphabetic characters,
+pub fn validate_channel_identifier(id: &str) -> Result<(), Error> {
+    validate_identifier(id, 8, 64)
```

Comment:
There is discrepancy between the [spec](https://github.com/cosmos/ibc/blob/master/spec/core/ics-024-host-requirements/README.md#paths-identifiers-separators) and the code here. The spec says channel ids must be 10 characters long or more but that would mean that `channel-0` is not a valid channel id. We should clarify that.

Reply:
Clarified offline. Let's update the comment above then